src/settings_type.h

```diff
@@ -77,6 +77,13 @@ enum ParticipateSurvey {
 	PS_YES,
 };
 
+/** Right-click to close window actions. */
+enum RightClickClose : uint8 {
+	RCC_NO_ACTION,
+	RCC_YES,
+	RCC_YES_EXCEPT_STICKY,
```

Comment:
These names are a bit unbalanced. `no-action` sounds good, but `yes` is then a bit weird. I would either expect no/yes, or some other wording for yes.

To make code more explicit, I don't mind the `RCC_NO_ACTION`. A suggestion could be `RCC_CLOSE_ALL` as the second option (and `RCC_CLOSE_EXCEPT_STICKY` for the third). But otherwise please make it `RCC_NO`. Either choice is fine by me.

The reason for this is that it might otherwise confuse future-us in a year or so, thinking that there was some special meaning here.